app.js

```diff
@@ -0,0 +1,42 @@
+var express = require('express');
+var app = express();
+
+// mongoose is just another node module
+var mongoose = require('mongoose');
+var db = mongoose.connect('mongodb://localhost/trucksAPI');
+var bodyParser = require('body-parser');
+
+var Truck = require('./models/truckModel');
+var idManager = require('./idManager');
+
+idManager.setIds(trucks.getTrucks());
```

Comment:
The trucks module doesn't exist, so it shouldn't be used here - causes the application to error out.